etc/project_status_definitions.yaml

```diff
@@ -51,6 +51,7 @@ step_completed_to_status:
     bioinformatics: DELIVERY
     Data Release EG 1.0: FINISHED
```

Comment:
I assume we still want this here to view older samples, which will have been run through Data Release EG 1.0?

Reply:
Yes that is correct